packages/ia-activity-indicator/src/ia-activity-indicator.js

```diff
@@ -0,0 +1,162 @@
+import { LitElement, html, css } from 'lit-element';
+
+const IAActivityIndicatorMode = Object.freeze({
+  processing: 'processing',
+  complete: 'complete',
+});
+
+// eslint-disable-next-line import/prefer-default-export
+class IAActivityIndicator extends LitElement {
+  static get properties() {
+    return {
+      mode: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.mode = IAActivityIndicatorMode.processing;
+  }
+
+  render() {
+    /* eslint-disable */
+    return html`
+      <div class="${this.mode}">
+        <svg
+          viewBox="0 0 120 120"
+          preserveAspectRatio="none"
+          version="1.1"
+          xmlns="http://www.w3.org/2000/svg"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
We can add `aria-labelledby` and corresponding `<title>` and `<desc>` elements to provide a text replacement and description of what is visually represented in this SVG object. There's an example in https://github.com/internetarchive/iaux/blob/feature-mobile-topnav/packages/mobile-top-nav/src/assets/img/icon-ellipses.js